nextbike/datapreparation/datapreparation.py

```diff
@@ -85,17 +85,17 @@ def data_preparation(df_original):
 
     print("Merge corresponding start and end...")
 
-    df_start = df_final[df_final["trip"] == "start"]
-    df_end = df_final[df_final["trip"] == "end"]
+    df_start = df_final[df_final["trip"] == "start"].drop("trip", axis=1)
+    df_end = df_final[df_final["trip"] == "end"].drop("trip", axis=1)
 
     df_end.reset_index(drop=True, inplace=True)
     df_start.reset_index(drop=True, inplace=True)
 
     df_merged = df_start.merge(df_end, left_on=df_start.index, right_on=df_end.index, suffixes=('_start', '_end'))
     df_merged.drop(["key_0",
-                    "trip_start",
                     "b_number_end",
-                    "trip_end"], axis=1, inplace=True)
+                    "Unnamed: 0_start",
+                    "Unnamed: 0_end"], axis=1, inplace=True)
```

Comment:
Are these "Unnamed 0" columns created during merge? otherwise they could also be dropped before merge like trip column

Reply:
Not quiet sure. I think save_csv also saves the index which in our dfs is not named. Therefore read_csv reads the saved file and there is a column with no name. This therefore is called Unnamed: 0. 
Solution would be: name our index column and while read_csv use index_col="Name" to use the Unnamed: 0 as the index...

Reply:
#48

Reply:
Ah nice